SEP/SEP-0007/sep-0007.md

```diff
@@ -95,33 +95,33 @@ FILTER EXISTS { BIND ( :e AS ?z )
 
 Both positions `AS ?z` and `SELECT ?x` must be variables.
 
-In the algebra, this affects
- * [`extend`](https://www.w3.org/TR/sparql11-query/#defn_extend) (related to the use of `AS` in SPARQL syntax)
- * [in line data](https://www.w3.org/TR/sparql11-query/#inline-data) (related to the use of `VALUES`)
+In the algebra, this affects —
+ * [`extend`](https://www.w3.org/TR/sparql11-query/#defn_extend) — related to the use of `AS` in SPARQL syntax
```

Comment:
This (before "related") is one of the few places an em-dash _might_ be used because it is inline (same line) text.

Reply:
The em-dash is appropriate here because the text that follows it is important to the text that precedes it. Parentheses are less appropriate here, for the same reason.